### PR TITLE
Update business logic for the flyover per UX feedback

### PR DESCRIPTION
**Pull request by MatthewPiatetsky**
Per UX feedback:
1. Default flyover to open
2. Make cookie that saves the open/close state course specific
3. Expire cookie after 4 hours

### CODE REVIEW

File: src/courseware/course/Course.jsx [hunk 1]
```diff
@@ -60,18 +60,18 @@ function Course({
...
       const exdate = new Date();
-      exdate.setDate(exdate.getDate() + exdays);
-      const cookieValue = escape(value) + ((exdays == null) ? '' : `; expires=${exdate.toUTCString()}`);
+      exdate.setHours(exdate.getHours() + 4);
+      const cookieValue = escape(value) + ((4 == null) ? '' : `; expires=${exdate.toUTCString()}`);
       document.cookie = `${cookieName}=${cookieValue};${cookieDomain}path=/`;
     };
     const isVisible = isREV1512FlyoverVisible();
-    setCookie('REV1512FlyoverVisible', !isVisible);
+    setCookie(`REV1512FlyoverVisible${courseId}`, !isVisible);
```
Comment (inventhouse) [inline src/courseware/course/Course.jsx:78]: How many cookies should we be creating?  Seems like the cookie-space could get crowded.

**Review by inventhouse — APPROVED** [review]

File: src/courseware/course/Course.jsx (hunk 1 above)
Comment (inventhouse) [reply src/courseware/course/Course.jsx:78]: (On the other hand, with a 4-hour expiration, maybe it won't be a problem)

File: src/courseware/course/Course.jsx [hunk 2]
```diff
@@ -60,18 +60,18 @@ function Course({
     const match = document.cookie.match(`${name}=([^;]*)`);
     return match ? match[1] : undefined;
   };
-  const [REV1512FlyoverVisible, setREV1512FlyoverVisible] = useState(getCookie('REV1512FlyoverVisible') === 'true');
+  const [REV1512FlyoverVisible, setREV1512FlyoverVisible] = useState(!(getCookie(`REV1512FlyoverVisible${courseId}`) === 'false'));
```
Comment (inventhouse) [inline src/courseware/course/Course.jsx]: Defaulting to open would be terrible on Mobile, since that takes over the entire screen

**Review by inventhouse — CHANGES_REQUESTED** [review]

File: src/courseware/course/Course.jsx (hunk 1 above)
Comment (MatthewPiatetsky) [reply src/courseware/course/Course.jsx:78]: i think it should be fine with a 4 hour expiration for the experiment

File: src/courseware/course/Course.jsx (hunk 2 above)
Comment (MatthewPiatetsky) [reply src/courseware/course/Course.jsx]: yeah i agree, updated

**Review by inventhouse — APPROVED** [review]

**Pull request merged**